components/Table/Table.vue

```diff
@@ -26,13 +37,40 @@ export default {
     items: {
       type: Array,
       required: true
+    },
+    showExpand: Boolean
+  },
+  data() {
+    return {
+      toggle: [],
+      expanded: []
+    };
+  },
+  computed: {
+    columnSpan() {
+      return this.showExpand ? this.headers.length + 1 : this.headers.length;
     }
   },
   methods: {
     textAlign(align) {
       return {
         "text-align": align || "left"
       };
+    },
+    expandDetail(id) {      
+      const index = this.toggle.indexOf(id);
+      if (index > -1) {
+        this.toggle.splice(index, 1);
+      } else {
+        this.toggle.push(id);
+      }
+
+      const expanded = this.expanded.find(e => e.id === id);
```

Comment:
what is this for? It doesn't seem to be used.

Reply:
Unused code, removed.